packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -1,103 +1,479 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/Chart.yaml packages/rancher-gatekeeper/charts/Chart.yaml
 --- packages/rancher-gatekeeper/charts-original/Chart.yaml
 +++ packages/rancher-gatekeeper/charts/Chart.yaml
-@@ -1,10 +1,16 @@
+@@ -1,6 +1,6 @@
  apiVersion: v1
  description: A Helm chart for Gatekeeper
--name: gatekeeper-operator
+-name: gatekeeper
 +name: rancher-gatekeeper
  keywords:
    - open policy agent
--version: v3.1.0-beta.7
-+version: 0.1.0
- home: https://github.com/open-policy-agent/gatekeeper
+ version: v3.1.0-beta.11
+@@ -8,3 +8,9 @@
  sources:
    - https://github.com/open-policy-agent/gatekeeper.git
- appVersion: v3.1.0-beta.7
```

Comment:
you could remove old version to clean things up since the chart hasn't gone live yet, but not necessary

Reply:
@cbron remove the old version as in remove the previous patch?

Reply:
I'll remove it.